src/sentry/static/sentry/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -476,11 +227,151 @@ const AddWidgetWrapper = styled('a')`
   justify-content: center;
 `;
 
-const WidgetGhost = styled('div')`
-  display: block;
-  position: absolute;
-  cursor: grabbing;
-  opacity: 0.8;
-  box-shadow: ${p => p.theme.dropShadowHeavy};
-  border-radius: ${p => p.theme.borderRadius};
-`;
+type SortableWidgetProps = {
+  widget: Widget;
+  dragId: string;
+  isEditing: boolean;
+  isDragging: boolean;
+  onDelete: () => void;
+  onEdit: () => void;
+};
+
+function SortableWidget(props: SortableWidgetProps) {
+  const {widget, dragId, isDragging, isEditing, onDelete, onEdit} = props;
+
+  const {
+    attributes,
+    listeners,
+    setNodeRef,
+    transform,
+    isDragging: currentWidgetDragging,
+  } = useSortable({
+    id: dragId,
+    transition: null,
+  });
+
+  useEffect(() => {
+    if (!currentWidgetDragging) {
+      return undefined;
+    }
+
+    document.body.style.cursor = 'grabbing';
```

Comment:
I would just pass the `cursor: grabbing` style in the sortable item, otherwise, if the parent component of the component where you want this cursor has a defined cursor, this will not work

Reply:
For manual dnd implementations, this is how I implemented it.

I took this idea from dndkit's example.  But I think it might be better to set the cursor on `document.body`.

When moving the mouse cursor fast enough such that the dragged item lags behind, then the mouse cursor style might be intermittently switched until the dragged item catches up to the mouse cursor.

Reply:
yes, I saw their example and tried it out but it didn't work well on the draggableList